tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -137,7 +137,7 @@ def _sync_rbds(self):
                     size=v['size'],
                     pool_id=pool_id,
                     flags=v['flags'],
-                    provisioned=self._to_bytes(v['provisioned']),
+                    provisioned=self._to_bytes(v.get("provisioned", 0)),
```

Comment:
do you really want to set the value to 0 if not found?
I think v.get("provisioned") will solve this issue as if "provisioned" is not found , value will be set to 'None'